sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -2390,3 +2395,41 @@ def _weights(dist):
 
     with pytest.raises(ValueError, match=msg):
         est.predict_proba([[1.1, 1.1]])
+
+
+def test_neighbor_classifiers_loocv():
+    """Check that `predict` and related functions work fine with X=None"""
+    X, y = datasets.make_blobs(n_samples=500, centers=5, n_features=2, random_state=0)
+
+    models = [
+        neighbors.KNeighborsClassifier(n_neighbors=10),
+        neighbors.RadiusNeighborsClassifier(radius=5.0),
+    ]
+
+    for knn in models:
+        loocv = cross_val_score(knn, X, y, cv=LeaveOneOut())
+
+        knn.fit(X, y)
+
+        assert np.all(loocv == (knn.predict(None) == y))
+        assert np.mean(loocv) == knn.score(None, y)
+        assert knn.score(None, y) < knn.score(X, y)
+
+
+def test_neighbor_regressors_loocv():
```

Comment:
I think that the name of this test does not quite match because we are still using models which are classifiers?